### PR TITLE
Cache player list

### PR DESCRIPTION
**Pull request by AriGute**
cache players list in localStorage and fix in tailwind css..

### CODE REVIEW

File: src/components/AboutPage/About.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const About = () => {
+	return (
+		<div className=' flex justify-center min-w-[300px]'>
+			<pre>
+				{`
+Welcome to TeamsGenerator, your ultimate solution for effortless team formation!
```
Comment (Ronielo) [inline src/components/AboutPage/About.js]: Save the string in a const outside the component

File: src/components/TeamGeneratorPage/ImportPlayerList.js [hunk 2]
```diff
@@ -12,6 +12,12 @@ const ImportPlayerList = ({ callback }) => {
 		callback(list);
 	}
 
+	useEffect(() => {
+		const playersList = getStoredPlayersList();
+		setPlayerList(playerList);
+	}, []);
+	window.addEventListener('clear', () => setPlayerList([]));
```
Comment (Ronielo) [inline src/components/TeamGeneratorPage/ImportPlayerList.js:19]: Why do you need this?

File: src/components/TeamGeneratorPage/TeamGenerator.js [hunk 3]
```diff
@@ -11,10 +12,17 @@ const TeamGenerator = () => {
 	};
 
 	function importPlayerList(importPlayers) {
-		const set = new Set([...playerList, ...importPlayers]);
-		setPlayerList([...set]);
+		const playersSet = new Set([...playerList, ...importPlayers]);
```
Comment (Ronielo) [inline src/components/TeamGeneratorPage/TeamGenerator.js:15]: Why do you need Set?

File: src/components/TeamGeneratorPage/TeamGenerator.js [hunk 4]
```diff
@@ -11,10 +12,17 @@ const TeamGenerator = () => {
...
+		const players = [...playersSet];
+		setStoredPlayersList(players);
+		setPlayerList(players);
 	}
 
+	useState(() => {
```
Comment (Ronielo) [inline src/components/TeamGeneratorPage/TeamGenerator.js:21]: What is that useState???

File: src/components/TeamGeneratorPage/TeamsDisplay.js [hunk 5]
```diff
@@ -15,6 +14,7 @@ const TeamsDisplay = ({ playerList }) => {
 
 	function clearButton() {
 		window.dispatchEvent(new Event('clear'));
```
Comment (Ronielo) [inline src/components/TeamGeneratorPage/TeamsDisplay.js:16]: Why the hell do you need an event listener??
If you need a state management use MobX

File: src/components/TeamGeneratorPage/TeamGenerator.js (hunk 3 above)
Comment (AriGute) [reply src/components/TeamGeneratorPage/TeamGenerator.js:15]: to remove duplicates players

File: src/components/TeamGeneratorPage/ImportPlayerList.js (hunk 2 above)
Comment (AriGute) [reply src/components/TeamGeneratorPage/ImportPlayerList.js:19]: to manage states and logic from different components

File: src/components/TeamGeneratorPage/TeamGenerator.js (hunk 4 above)
Comment (AriGute) [reply src/components/TeamGeneratorPage/TeamGenerator.js:21]: fixed in https://github.com/AriGute/TeamsGenerator/pull/3

File: src/components/TeamGeneratorPage/TeamsDisplay.js (hunk 5 above)
Comment (AriGute) [reply src/components/TeamGeneratorPage/TeamsDisplay.js:16]: to manage states and logic from different components

**Pull request closed without merging**